src/vs/workbench/services/positronConnections/browser/positronConnectionsInstance.ts

```diff
@@ -191,23 +248,25 @@ export class PositronConnectionsInstance extends BaseConnectionsInstance impleme
 export class DisconnectedPositronConnectionsInstance extends BaseConnectionsInstance implements IPositronConnectionInstance {
 	constructor(
 		metadata: ConnectionMetadata,
-		readonly onDidChangeDataEmitter: Emitter<void>,
 		readonly runtimeSessionService: IRuntimeSessionService,
+		readonly connectionsService: IPositronConnectionsService
 	) {
 		super(metadata);
 	}
 
 	readonly kind: string = 'database';
 	readonly expanded: boolean | undefined = false;
 	readonly active: boolean = false;
+	readonly onDidChangeStatus = Event.None;
+	readonly onToggleExpandEmitter = new Emitter<string>();
```

Comment:
```
readonly onToggleExpandEmitter = this._register(new Emitter<string>());
```

Reply:
Resolved in fe6a0d009ddab8ff0dc5ac6b3cfd3145ec8cdd97